db.json

```diff
@@ -1,46 +1,118 @@
 {
   "tasks": [
     {
-      "id": 1,
-      "title": "json-server",
-      "description": "typicode",
-      "done": false
+      "title": "Random Cat Fact",
+      "description": "A cat's whiskers are thought to be a kind of radar, which helps a cat gauge the space it intends to walk through.",
+      "done": false,
+      "id": 8
```

Comment:
You should wipe clean your db and not commit it with data from the last use, as this affects the user experience.

Reply:
We added the file to the gitignore